### DISCUSSION TITLE
не получается принять заявку в друзья

### DISCUSSION THREAD

**Discussion by DizzieDawg** [stoatchat/discussions#757] [General]
мой друг в revolt столкнулся с проблемой что он не может принять мой запрос в друзья + он не может кинуть мне заявку в друзья
ошибка либо 409 либо 404
в добавок к этому почти не возможно принять заявку

**Comment by 1enify** [comment]
Здравствуйте.
По поводу ошибки 404: проверьте правильность написания никнэйма(должно быть `никнэйм#тэг`)
Ошибка 409 возможно значит "Unprocessable entity"(как перевести это я, увы, не знаю). В качестве решения предлагают перезапустить/обновить клиент.
За последующей поддержкой можно обратиться [сюда](https://rvlt.gg/Z5NMY79a) (оффициальное сообщество Revolt, писать на английском)
